data/pigui/game.lua

```diff
@@ -16,6 +16,9 @@ local icons = ui.theme.icons
 local reticuleCircleRadius = math.min(ui.screenWidth, ui.screenHeight) / 13
 local reticuleCircleThickness = 2.0
 
+-- settings
+local ASTEROID_RADIUS = 1500 -- rocky planets smaller than this are considered an asteroid, not a planet
```

Comment:
Maybe @fluffyfreak knows a better way/place to define this, since he did some asteroid work on a branch of his? Anyway, just wanted his eyes on that line. I have no complaint.